.gitignore

```diff
@@ -1,3 +1,5 @@
 env
 my_vars.yml
 secret.yml
+secret.ocp3.yml
+secret.ocp4.yml
```

Comment:
@jmontleon can you help me understand the need for secret.ocp3.yml and secret.ocp4.yml

Reply:
Please ignore me...sorry I thought this PR introduced them, I see you are just fixing it so they arent committed by accident, thank you!

https://github.com/konveyor/mig-agnosticd/pull/196 has the info I was looking for.